app/views/reports/regions/_hypertension_patient_coverage.html.erb

```diff
@@ -0,0 +1,85 @@
+<div class="d-flex ai-center jc-between">
+  <p class="mb-8px">
+    Hypertension patient coverage
+    <span class="p-relative t-1px">
+      <i
+        class="fas fa-question-circle fs-14px c-grey-dark"
+        data-toggle="tooltip"
+        data-placement="right"
+        data-trigger="hover click"
+        data-html="true"
+        title="<%= t('hypertension_patient_coverage_copy') %>"
+      >
+      </i>
+    </span>
+  </p>
+  <p class="mb-8px fw-bold">
+    <% if @region.district_region? ||
+      @region.state_region? && @is_population_available_for_all_districts %>
+      <span class="c-green-dark">
+        <%= number_to_percentage(@hypertension_patient_coverage, precision: 0) %>
+      </span>
+    <% else %>
+      ???
+    <% end %>
+  </p>
+</div>
+<div class="d-flex h-8px mb-8px">
+  <div class="br-tl-4px br-bl-4px <% if !@hypertension_patient_coverage.zero? || !@hypertension_patient_coverage %>bg-grey-light<% else %>bg-green-dark<% end %>" style="width: <% if !@hypertension_patient_coverage.zero? || !@hypertension_patient_coverage %>5%<% else %><%= number_to_percentage(@hypertension_patient_coverage, precision: 0) %><% end %>;"></div>
```

Comment:
This is really hard to read -- mind adding some new lines? Then I think we can find some ways to clean it up.